spec/legacy/openstack_handle/handle_spec.rb

```diff
@@ -0,0 +1,191 @@
+require 'manageiq/providers/openstack/legacy/openstack_handle/handle'
+require 'fog/openstack'
```

Comment:
I suspect this is here for the `and_raise(Fog::Network::OpenStack::NotFound)` in the test, but should this be required up front somewhere rather than in the test?